frontend/dashboard/pages/CreateService/CreateService.tsx

```diff
@@ -110,6 +112,11 @@ export const CreateService = ({ user, organizations }: CreateServiceProps): JSX.
     return isOrgValid && isRepoNameValid;
   };
 
+  const validateTextValue = (event: ChangeEvent<HTMLInputElement>) => {
+    const { isValid: isRepoNameValid } = validateRepoName(event.target.value);
+    if (isRepoNameValid) setFormError(initialFormError);
+  };
+
```

Comment:
Nice work 👍 I tested it locally and can confirm that it works as expected, but I now noticed that it feels a bit strange to have to click the button to see the error:

https://github.com/Altinn/altinn-studio/assets/24462611/99f2d5df-1e53-456d-ab6d-0da8e9392800

Now that we validate the field on the onChange event, I suggest displaying the error when the value changes, something like this:
```suggestion
  const validateTextValue = (event: ChangeEvent<HTMLInputElement>) => {
    const { errorMessage: repoNameErrorMessage, isValid: isRepoNameValid } = validateRepoName(
      event.target.value,
    );
    setFormError((previous) => ({
      ...previous,
      repoName: isRepoNameValid ? '' : repoNameErrorMessage,
    }));
  };
```

https://github.com/Altinn/altinn-studio/assets/24462611/cb13e1f2-4d21-4452-b1d0-4dc4afeb9466

Reply:
Great suggestion! I totally agree 😄